.flutter-plugins

```diff
@@ -1,55 +1,55 @@
 # This is a generated file; do not edit or check into version control.
```

Comment:
We need to put following files in `.gitignore` file

- `.flutter-plugins`
- `.flutter-plugins-dependencies`